### DISCUSSION TITLE
Should everything here need to be Open Source, or just "anything useful" even if closed source, or even paid?

### DISCUSSION THREAD

**Discussion by jordn** [humanloop/awesome-chatgpt#68] [Q&A]
Trying to come up with a policy of what should be included.

Please let me know your thoughts!

**Comment by TFWol** [comment]
Probably anything useful, but have it marked so people know.

Especially since this repo is at the top results when searching for things like **ChatGPT** and **Awesome ChatGPT**. 

That means more exposure which means more pull requests/issues to sift through. 
Organizing and labeling becomes more important the more data there is.

Having a template would help.
